common/Tests/Framework/src/TelemetryValidation/CommonValidator.ts

```diff
@@ -0,0 +1,24 @@
+import { ITypeValidator } from './ITypeValidator';
```

Comment:
Effectively copied from AISKU to the shared framework -- I didn't refactor the AISKU tests to use the framework as part of this PR as this is already larger than I'd like.